nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestPutSQL.java

```diff
@@ -173,12 +192,10 @@ public void testFailInMiddleWithBadStatement() throws InitializationException, P
         runner.enqueue("INSERT INTO PERSONS_AI".getBytes()); // intentionally wrong syntax
         runner.enqueue("INSERT INTO PERSONS_AI (NAME, CODE) VALUES ('Tom', 3)".getBytes());
         runner.enqueue("INSERT INTO PERSONS_AI (NAME, CODE) VALUES ('Harry', 44)".getBytes());
-        runner.run();
-
-        runner.assertTransferCount(PutSQL.REL_FAILURE, 1);
```

Comment:
Should we be asserting something different here? Doesn't seem like we'd need this test otherwise, since we're not running the flow or verifying any results.

Reply:
the common method `testFailInMiddleWithBadStatement` is called by `testFailInMiddleWithBadStatementAndSupportTransaction` and `testFailInMiddleWithBadStatementAndNotSupportTransaction`.These two unit test have the the same properties except `Support Fragmented Transactions`. So I exact the common code. Thanks for reviewing.